data/json/items/gun/44.json

```
@@ -1,42 +1,45 @@
 [
   {
-    "id": "deagle_44",
+    "id": "modular_deagle",
     "looks_like": "glock_17",
     "type": "GUN",
     "reload_noise_volume": 10,
-    "name": { "str_sp": "Desert Eagle .44" },
-    "description": "One of the most recognizable handguns due to its popularity in movies and video games, the Desert Eagle is better known for its menacing appearance than its performance.  It's a large pistol, but its heavy weight reduces recoil.",
-    "weight": "1700 g",
-    "volume": "617 ml",
+    "name": { "str_sp": "Desert Eagle pistol" },
+    "description": "An icon of popular culture more than a practical weapon, this massive pistol can shoot a variety of magnum handgun rounds depending on the conversion kit and magazine used.",
+    "weight": "1407 g",
+    "volume": "671 ml",
     "longest_side": "29 cm",
-    "barrel_length": "152 mm",
-    "price": 87500,
+    "//": "Numbers for weight, volume, and length calculations are based upon weapon without caliber conversion. The difference will be made up for with integrated stats from the installable caliber conversion.",
+    "price": 160000,
     "price_postapoc": 2750,
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "modular_deagle",
+        "name": { "str": "Desert Eagle Mark XIX" },
+        "description": "Needing little introduction, the Desert Eagle's impact on popular culture is almost as massive as the pistol itself.  The newest Mark XIX model introduced a more modern optics rail.  It was sold in multiple hard-hitting magnum calibers and can swap between them using conversion kits and the appropriate magazine."
+      },
```

```suggestion
      {
        "id": "modular_deagle",
        "name": { "str": "Desert Eagle Mark XIX" },
        "description": "Needing little introduction, the Desert Eagle's impact on popular culture is almost as massive as the pistol itself.  The newest Mark XIX model introduced a more modern optics rail.  It was sold in multiple hard-hitting magnum calibers and can swap between them using conversion kits and the appropriate magazine.",
        "weight": 10000
      },
```
You probably want to put some absurdly high weight here, so probability to spawn `modular_deagle_gold` is small enough